Packs/EWS/TestPlaybooks/playbook-EWS_O365_send_mail_test.yml

```diff
@@ -0,0 +1,745 @@
+id: EWS_O365_send_mail_test
```

Comment:
Don't we have an existing TPB?

Reply:
We do, but this command has some functionality to check, and I thought it would be better to separate it